### PR TITLE
Fixed issue #16373 : Bootstrap Buttons not aligned when filtered

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: id to the wrapper, the id is hidden

### CODE REVIEW

File: themes/question/bootstrap_buttons/survey/questions/answer/listradio/assets/scripts/bootstrapbuttons.js [hunk 1]
```diff
@@ -9,5 +9,4 @@ function doBootstrapRadio(){
             $("#answer" + name + "othertext").val('').trigger("change");
         }
     });
-
```
Comment (olleharstedt) [inline themes/question/bootstrap_buttons/survey/questions/answer/listradio/assets/scripts/bootstrapbuttons.js:12]: For the future, please try to refrain from edits like these.

File: themes/question/bootstrap_buttons/survey/questions/answer/multiplechoice/config.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="UTF-8"?>
```
Comment (olleharstedt) [inline themes/question/bootstrap_buttons/survey/questions/answer/multiplechoice/config.xml:1]: Also empty.

**Review by olleharstedt — APPROVED** [review]

File: themes/question/bootstrap_buttons/survey/questions/answer/multiplechoice/config.xml (hunk 2 above)
Comment (Shnoulle) [reply themes/question/bootstrap_buttons/survey/questions/answer/multiplechoice/config.xml:1]: Remove space at end ? Unsure (don't remind)

Comment (olleharstedt) [reply themes/question/bootstrap_buttons/survey/questions/answer/multiplechoice/config.xml:1]: Can be Microsoft vs Unix line ending.

Comment (Shnoulle) [reply themes/question/bootstrap_buttons/survey/questions/answer/multiplechoice/config.xml:1]: Can not : else have it at all lines. Start of the line ? BOM ?

**Pull request merged**